scripts/pkgs/macos-11.txt

```diff
@@ -0,0 +1,4 @@
+astyle
```

Comment:
Surprisingly few deps, once SPDK is not taken into consideration.
What about ``clang``? I guess that comes with the user-land? No need to specify installation of it?
I guess ``astyle`` can be removed at this point?

Reply:
A working installation of `brew` assumes Xcode and its command line tools are installed. This is what supplies `clang`, and likely a lot of other smaller dependencies.

I've removed `astyle`.